components/Aside.js

```diff
@@ -17,7 +17,11 @@ const FileList = ({ title, files, packageName }) => html`
         <li key=${x.url} data-test="Item">
           ${FileIcon}
           <${Link} href=${x.url.replace('https://unpkg.com/', '/?')}>
-            ${x.url.replace(`https://unpkg.com/`, '').replace(packageName, '')}
+            <p>
```

Comment:
How come we need a `p` inside an `a`. Not a blocker, just curious.

Reply:
Oops sorry, had merged this just a few seconds before you saw this. Good spot! I assumed this was a "hack" to get the wrapping working as expected in the aside @kiraarghy

Reply:
I don't often review PRs.. but when I do 👀👀👀 

![image](https://user-images.githubusercontent.com/1457604/59115280-df5f7c80-8940-11e9-8f42-6942a28eb1c6.png)